src/libData/AccountData/AccountStoreSC.h

```diff
@@ -46,10 +46,10 @@ class AccountStoreAtomic
   GetAddressToAccount();
 };
 
+enum INVOKE_TYPE { CHECKER, RUNNER_CREATE, RUNNER_CALL };
```

Comment:
I tried this, but turns out, uses such as this: https://github.com/Zilliqa/Zilliqa/home/vaivaswatha/d/Zilliqa/src/libData/AccountData/AccountStore.cpp#L567 won't work because it becomes private. I can either redo this change (i.e., move it out of the class) or make it public I guess.

Reply:
Oh okay, I did not notice the migration needed it. You can ignore this comment then.